packages/formatter-html/src/views/partials/scan-category-summary.ejs

```diff
@@ -2,15 +2,15 @@
     <div class="rule-tile">
         <img src="<%= category.image %>" class="rule-icon" alt="" role="presentation">
         <div class="rule-tile__info">
-            <p class="rule-tile__category"><%= category.name.toUpperCase() %></p>
+            <p class="rule-tile__category"><%= category.localizedName.toLocaleUpperCase(); %></p>
```

Comment:
I think we should tackle part of webhintio/webhint.io#657 here and drop converting the name to upper case. As that issue notes, this doesn't always work correctly for localized strings and we should just used the sentence-cased category name instead.